sharding/indexHashedNodesCoordinatorWithRater.go

```diff
@@ -55,6 +55,23 @@ func (ihgs *indexHashedNodesCoordinatorWithRater) SetNodesPerShards(
 	return ihgs.expandAllLists(epoch)
 }
 
+// ComputeLeaving - computes the validators that have a threshold below the minimum rating
+func (ihgs *indexHashedNodesCoordinatorWithRater) ComputeLeaving(validators []Validator) ([]Validator, error) {
+	leavingValidators := make([]Validator, 0)
+	for _, validator := range validators {
+		pk := validator.PubKey()
+		rating := ihgs.GetRating(string(pk))
+		chances := ihgs.GetChance(rating)
+		log.Trace("Compute leaving", "pk", pk, "chances", chances)
+		if chances == 0 {
```

Comment:
shouldn't be like chances < MinChances ?

Reply:
It works both ways, below a ratingThreshold you get  0 chances and you should still get some chances, because there are not enough validators in the lists. Refactored to use minChances